src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/BlazorWeb-CSharp/Program.cs

```diff
@@ -14,6 +14,8 @@
 using BlazorWeb_CSharp.Data;
 #endif
 
+System.AppContext.SetSwitch("Microsoft.AspNetCore.Components.Endpoints.NavigationManager.DisableThrowNavigationException", true);
```

Comment:
Precisely being able to put a `Directory.build.props` and defining it there is what I was going for with avoiding this